scripts/install_whonix_qubes/README.md

```diff
@@ -0,0 +1,75 @@
+# Install Haveno on Qubes/Whonix
+
+
+After you already have [`Qubes`](https://www.qubes-os.org/downloads) or [`Whonix`](https://www.whonix.org/wiki/Download) installed:
+
+1. Download [scripts](https://github.com/PromptPunksFauxCough/haveno/tree/install_whonix_qubes/scripts/install_whonix_qubes/scripts).
```

Comment:
This should be changed to the haveno-dex repo, so the link will be valid when merged.